.github/workflows/build.yml

```diff
@@ -132,7 +132,8 @@ jobs:
     - name: Install Dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y software-properties-common clang-format cppcheck luajit-5.1-dev luarocks mariadb-server mariadb-client libmariadb-dev-compat
+        sudo apt-get install -y software-properties-common clang-format cppcheck luajit-5.1-dev luarocks mariadb-server mariadb-client libmariadb-dev-compat python3.7
```

Comment:
Is 3.7 the intended version? I thought we required greater than that now?

Reply:
It's copy/paste from the other CI entries, I don't think we have a hard reliance on any Python 3.x version. Happy to have CI use latest